docs/security/clerk-csp.mdx

```diff
@@ -48,3 +50,9 @@ const nextConfig = {
 
 module.exports = nextConfig;
 ```
+
+Some notes about the `unsafe-eval` and `unsafe-inline` directives:
+
+- Within `script-src`, `unsafe-eval` is a [requirement for next.js](https://nextjs.org/docs/app/building-your-application/configuring/content-security-policy#without-nonces) to run in development mode. *For production environment CSPs, it should be removed*.  
+- Within `script-src`, `unsafe-inline` is a requirement for next.js in both dev and prod environments if you're using app router. If you are using pages router, it should be removed.
+- Within `style-src`, `unsafe-inline` is a requirement for Clerk's components. Removing this requirement is on our roadmap - please [reach out to support](https://clerk/.com/support) if you'd like to see this implemented sooner.
```

Comment:
Should we point them to the feedback portal instead of support@?

```suggestion
- Within `style-src`, `unsafe-inline` is a requirement for Clerk's components to inject their styles. Removing this requirement is on our roadmap - please [reach out to support](https://clerk/.com/support) if you'd like to see this implemented sooner.
```

Reply:
i feel like i'd prefer to start with support here but if we get a lot of them will switch over